data/presets/shop/pawnbroker.json

```diff
@@ -7,5 +7,9 @@
     "tags": {
         "shop": "pawnbroker"
     },
-    "name": "Pawn Shop"
+    "name": "Pawn Shop",
+    "aliases": [
+        "Pawnshop",
```

Comment:
[“Pawnshop” looks like a winner](https://books.google.com/ngrams/graph?content=pawn+shop%2Cpawnshop%2Cpawn+store%2Cpawnbroker&year_start=1800&year_end=2019&corpus=28&smoothing=3&direct_url=t1%3B%2Cpawn%20shop%3B%2Cc0%3B.t1%3B%2Cpawnshop%3B%2Cc0%3B.t1%3B%2Cpawn%20store%3B%2Cc0%3B.t1%3B%2Cpawnbroker%3B%2Cc0) of late.